python/cudf/cudf/io/parquet.py

```diff
@@ -625,14 +626,17 @@ def _parquet_to_frame(
             else:
                 # Not building categorical columns, so
                 # `value` is already what we want
+                _value = value
+                if pd.isna(value):
+                    _value, _len = [value] * _len, None
                 if partition_meta is not None:
                     dfs[-1][name] = as_column(
-                        value,
-                        length=len(dfs[-1]),
+                        _value,
+                        length=_len,
                         dtype=partition_meta[name].dtype,
                     )
```

Comment:
@galipremsagar - Do you know the most efficient way to construct a completely-null column in cudf?

Reply:
Yup, you can do `cudf.core.column.column_empty(row_count=..., dtype=..., masked=True)`

Reply:
or if you want to fill with a specific value, we have `cudf.core.column.full(size=..., fill_value=..., dtype=...)`

Reply:
Thanks!